examples/electron/app.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { render } from 'react-dom'
+
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/extensions */
```

Comment:
why are all these necessary? Can you just disable all linting for this one line?